### ISSUE TITLE
BIG BUG: Stylish broke the display of Devtools in Firefox 50

### ISSUE THREAD

**Issue by mikhoul** [stylish-userstyles/stylish#301]
Stylish broke the display of Devtools in Firefox 50, it put a annoying dark gray bar at the bottom of the devtool.
![GitHub Logo](http://i.imgur.com/Q47Ekd5.png)

When you disable Stylish it goes away.

@JasonBarnabe 

Regards :octocat:

**Comment by polyzen** [comment]
Is this even with styles off? I don't have the gray bar.

**Comment by mikhoul** [comment]
@polyzen   Yes !   Even with all my style disabled individually like this: 
![GitHub Logo](http://i.imgur.com/uH4yCn1.png)

But if I disable **completely** Stylish vis "Disable all Scripts" it goes away. 

That's why it is so strange  :octocat:

**Comment by auscompgeek** [comment]
I can't reproduce this either. Are you _completely_ sure it's not caused by a chrome userstyle?

**Comment by mikhoul** [comment]
Let me check double check and I will be back at the beginning of the week with an answer.

Regards :octocat:
